scipy/signal/_peak_finding.py

```diff
@@ -1036,7 +1036,7 @@ def _identify_ridge_lines(matr, max_distances, gap_thresh):
     ----------
     Bioinformatics (2006) 22 (17): 2059-2065.
     :doi:`10.1093/bioinformatics/btl355`
-    http://bioinformatics.oxfordjournals.org/content/22/17/2059.long
+    https://academic.oup.com/bioinformatics/article/22/17/2059/274284
```

Comment:
Original link redirects to here not sure if this should be changed at this time (happy to pull this out and include in a later PR if we care)

Reply:
Looks fine to me